Makefile.PL

```diff
@@ -22,6 +22,7 @@ requires
   'Log::Any'                    => 0,
   'Log::Any::Adapter::Dispatch' => 0,
   'Log::Dispatch'               => 0,
+  'LWP::UserAgent'              => 0,
```

Comment:
Do you have a comment for this change? I do not see that it is related to the rest.

Reply:
`zmb` nees it.